app/Http/Controllers/Api/MeController.php

```diff
@@ -0,0 +1,23 @@
+<?php namespace Symposium\Http\Controllers\Api;
+
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\Auth;
+use LucaDegasperi\OAuth2Server\Facades\AuthorizerFacade as Authorizer;
+use Symposium\ApiResources\Me;
+use User;
+
+class MeController extends BaseController
+{
+    public function index()
+    {
+        $me = new Me(User::find(Authorizer::getResourceOwnerId()));
```

Comment:
Haha yah I think it'll probably end up as "currentApiUser" or something equally boring